.gitignore

```diff
@@ -3,7 +3,6 @@
 **/.DS_Store
 **/.idea
 **/.ipynb_checkpoints
-**/.vscode
```

Comment:
We can always force-add files. Maybe we should keep this?

Reply:
Yeah... This dir has a fair bit of nuance around what's acceptable here ([ref](https://www.toptal.com/developers/gitignore/api/visualstudiocode) [ref](https://github.com/microsoft/vscode/issues/15909)) so block-by-default seems like a good idea.